### ISSUE TITLE
Unify learning_rate parameters across algorithms

### ISSUE THREAD

**Issue by Lumik7** [collinleiber/ClustPy#14]
Currently each deep clustering algorithm has an individually named learning rate parameter, e.g., DEC has `dec_learning_rate`. Change this to `learning_rate` to be consistent accross algorithms.
